pokemongo_bot/cell_workers/utils.py

```diff
@@ -82,7 +82,9 @@ def format_dist(distance, unit):
 
 def format_time(seconds):
     # Return a string displaying the time given as seconds or minutes
-    if seconds <= 1.0:
+    if seconds <= 0.0:
+        return '{:.2f} seconds'.format(seconds)
+    elif seconds <= 1.0:
```

Comment:
this code doesn't seem to be tied to the PR purpose, but as a point of feedback, it seems the intent here is that if `seconds` is exactly equal to 1 it should be "second", but when less than 1 it should be seconds. i.e. `1.0 second` and `0.99 seconds`